invenio_rdm_records/marshmallow/fields.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+# Copyright (C) 2020 Northwestern University.
+#
+# Invenio-RDM-Records is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Custom provided fields."""
+from datetime import date
+
+from edtf.parser.grammar import ParseException, level0Expression
+from flask_babelex import lazy_gettext as _
+from marshmallow import fields, missing
+
+
+class EDTFLvl0DateString(fields.Str):
```

Comment:
Comment: Likely the marshmallow field could be moved to Invenio-Records-REST later to have a reusable field for other Invenio installations.

Reply:
Sure! Made issue for it in invenio-records-rest: https://github.com/inveniosoftware/invenio-records-rest/issues/293